docs/usage.rst

```diff
@@ -323,6 +323,21 @@ object in-place and returns no result. A warning is emitted if your
 ``modifier`` returns a non-None result that is not the same object as the
 input.
 
+Using custom certificates
+-------------------------
+
+If you need to use custom certificates in your ``pystac-client`` requests, you can
+customize the :class:`StacApiIO<pystac_client.stac_api_io.StacApiIO>` instance before
+creating your :class:`Client<pystac_client.Client>`.
+
+.. code-block:: python
+
+    >>> from pystac_client.stac_api_io import StacApiIO
+    >>> from pystac_client.client import Client
+    >>> stac_api_io = StacApiIO()
+    >>> stac_api_io.session.verify = "/path/to/certfile"
+    >>> client = Client.from_file("https://planetarycomputer.microsoft.com/api/stac/v1", stac_io=stac_api_io)
```

Comment:
Is there a reason why we are using `Client.from_file()` rather than `Client.open()`?

Reply:
`open` doesn't take a `stac_io`: https://pystac-client.readthedocs.io/en/stable/api.html#client. Which seems like an inconsistency to me -- maybe it'd be worth adding `stac_io` to `open`? What do you think?

Reply:
It looks like it does:
https://github.com/stac-utils/pystac-client/blob/7d589e802eba2499cfd97c2948a2ed79856937fd/pystac_client/client.py#L90-L101

Reply:
Yup, looks like it's unreleased, was added here: https://github.com/stac-utils/pystac-client/pull/372. I'll update this example to use `open`.